cmake/Pcsx2Utils.cmake

```diff
@@ -125,7 +125,7 @@ macro(add_pcsx2_plugin lib srcs libs flags)
         target_link_libraries(${lib} "${USER_CMAKE_LD_FLAGS}")
     endif(NOT USER_CMAKE_LD_FLAGS STREQUAL "")
     if(PACKAGE_MODE)
-        install(TARGETS ${lib} DESTINATION ${PLUGIN_DIR})
+        install(TARGETS ${lib} DESTINATION ${CMAKE_INSTALL_LIBDIR}/PCSX2)
```

Comment:
Are you sure auto discovery of plugin works like this? What about local installations

Reply:
Can you elaborate what you mean by local installation? I tested this by applying the patch to my pcsx2 git slackbuild and installing the resulting package.

Everything seems to work for me.

Reply:
```
[nix-shell:~/Documents/projects/programming/EMULATORS/pcsx2]$ ls bin 
bios       cheats_ws.zip  GameIndex.dbf             Langs     PCSX2                   plugins              shaders
cheats     docs           GSdx_opengl_debug_hw.txt  logs      PCSX2_keys.ini.default  portable.ini         snaps
cheats_ws  dumps          inis                      memcards  PCSX2-linux.sh          run_spu2_replay.cmd  sstates
```

My make install drops everything into this folder in a dev environment, how would this change?

Reply:
Did you use `-DPACKAGE_MODE=ON`? These are all conditionally used only when that is enabled.

Here are the contents of my pcsx2 install.
```
usr/
usr/doc/
usr/doc/pcsx2-2020.12.01_cd9a0cd/
usr/doc/pcsx2-2020.12.01_cd9a0cd/CONTRIBUTING.md
usr/doc/pcsx2-2020.12.01_cd9a0cd/COPYING.GPLv2
usr/doc/pcsx2-2020.12.01_cd9a0cd/COPYING.GPLv3
usr/doc/pcsx2-2020.12.01_cd9a0cd/COPYING.LGPLv2.1
usr/doc/pcsx2-2020.12.01_cd9a0cd/COPYING.LGPLv3
usr/doc/pcsx2-2020.12.01_cd9a0cd/Configuration_Guide.pdf
usr/doc/pcsx2-2020.12.01_cd9a0cd/PCSX2_FAQ.pdf
usr/doc/pcsx2-2020.12.01_cd9a0cd/README.md
usr/doc/pcsx2-2020.12.01_cd9a0cd/pcsx2.SlackBuild
usr/games/
usr/games/PCSX2
usr/games/pcsx2_GSReplayLoader
usr/lib64/
usr/lib64/games/
usr/lib64/games/PCSX2/
usr/lib64/games/PCSX2/libGSdx-AVX2.so
usr/lib64/games/PCSX2/libGSdx-SSE4.so
usr/lib64/games/PCSX2/libGSdx.so
usr/lib64/games/PCSX2/libonepad-legacy.so
usr/lib64/games/PCSX2/libonepad.so
usr/man/
usr/man/man1/
usr/man/man1/PCSX2.1.gz
usr/share/
usr/share/applications/
usr/share/applications/PCSX2.desktop
usr/share/games/
usr/share/games/PCSX2/
usr/share/games/PCSX2/GameIndex.dbf
usr/share/games/PCSX2/cheats_ws.zip
usr/share/locale/
usr/share/locale/ar_SA/
usr/share/locale/ar_SA/LC_MESSAGES/
usr/share/locale/ar_SA/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/ar_SA/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/ca_ES/
usr/share/locale/ca_ES/LC_MESSAGES/
usr/share/locale/ca_ES/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/ca_ES/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/cs_CZ/
usr/share/locale/cs_CZ/LC_MESSAGES/
usr/share/locale/cs_CZ/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/cs_CZ/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/da_DK/
usr/share/locale/da_DK/LC_MESSAGES/
usr/share/locale/da_DK/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/da_DK/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/de_DE/
usr/share/locale/de_DE/LC_MESSAGES/
usr/share/locale/de_DE/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/de_DE/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/es_ES/
usr/share/locale/es_ES/LC_MESSAGES/
usr/share/locale/es_ES/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/es_ES/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/fi_FI/
usr/share/locale/fi_FI/LC_MESSAGES/
usr/share/locale/fi_FI/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/fi_FI/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/fr_FR/
usr/share/locale/fr_FR/LC_MESSAGES/
usr/share/locale/fr_FR/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/fr_FR/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/hr_HR/
usr/share/locale/hr_HR/LC_MESSAGES/
usr/share/locale/hr_HR/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/hr_HR/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/hu_HU/
usr/share/locale/hu_HU/LC_MESSAGES/
usr/share/locale/hu_HU/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/hu_HU/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/id_ID/
usr/share/locale/id_ID/LC_MESSAGES/
usr/share/locale/id_ID/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/id_ID/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/it_IT/
usr/share/locale/it_IT/LC_MESSAGES/
usr/share/locale/it_IT/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/it_IT/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/ja_JP/
usr/share/locale/ja_JP/LC_MESSAGES/
usr/share/locale/ja_JP/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/ja_JP/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/ko_KR/
usr/share/locale/ko_KR/LC_MESSAGES/
usr/share/locale/ko_KR/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/ko_KR/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/lt_LT/
usr/share/locale/lt_LT/LC_MESSAGES/
usr/share/locale/lt_LT/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/lt_LT/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/ms_MY/
usr/share/locale/ms_MY/LC_MESSAGES/
usr/share/locale/ms_MY/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/ms_MY/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/nb_NO/
usr/share/locale/nb_NO/LC_MESSAGES/
usr/share/locale/nb_NO/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/nb_NO/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/nl_NL/
usr/share/locale/nl_NL/LC_MESSAGES/
usr/share/locale/nl_NL/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/nl_NL/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/pl_PL/
usr/share/locale/pl_PL/LC_MESSAGES/
usr/share/locale/pl_PL/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/pl_PL/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/pt_BR/
usr/share/locale/pt_BR/LC_MESSAGES/
usr/share/locale/pt_BR/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/pt_BR/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/ru_RU/
usr/share/locale/ru_RU/LC_MESSAGES/
usr/share/locale/ru_RU/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/ru_RU/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/sv_SE/
usr/share/locale/sv_SE/LC_MESSAGES/
usr/share/locale/sv_SE/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/sv_SE/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/th_TH/
usr/share/locale/th_TH/LC_MESSAGES/
usr/share/locale/th_TH/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/th_TH/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/tr_TR/
usr/share/locale/tr_TR/LC_MESSAGES/
usr/share/locale/tr_TR/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/tr_TR/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/zh_CN/
usr/share/locale/zh_CN/LC_MESSAGES/
usr/share/locale/zh_CN/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/zh_CN/LC_MESSAGES/pcsx2_Main.mo
usr/share/locale/zh_TW/
usr/share/locale/zh_TW/LC_MESSAGES/
usr/share/locale/zh_TW/LC_MESSAGES/pcsx2_Iconized.mo
usr/share/locale/zh_TW/LC_MESSAGES/pcsx2_Main.mo
usr/share/pixmaps/
usr/share/pixmaps/PCSX2.xpm
```
Here are my cmake arguments.
```
    -DCMAKE_C_FLAGS="$SLKCFLAGS" \
    -DCMAKE_CXX_FLAGS="$SLKCFLAGS" \
    -DCMAKE_INSTALL_PREFIX=/usr \
    -DCMAKE_INSTALL_LIBDIR=lib${LIBDIRSUFFIX}/games \
    -DCMAKE_INSTALL_BINDIR=games \
    -DCMAKE_INSTALL_MANDIR=man \
    -DCMAKE_INSTALL_DOCDIR=doc/$PRGNAM-$VERSION \
    -DCMAKE_INSTALL_DATADIR=share/games \
    -DBUILD_REPLAY_LOADERS=ON \
    -DGTK2_API=ON \
    -DXDG_STD=ON \
    -DOPTIMIZATION_FLAG='' \
    -DPACKAGE_MODE=ON \
    -DDISABLE_ADVANCE_SIMD=ON \
    -DDISABLE_BUILD_DATE=ON \
    -DDISABLE_PCSX2_WRAPPER=ON \
    -DDISABLE_SETCAP=ON \
    -DENABLE_TESTS=OFF \
    -DUSE_ASAN=$ASAN \
    -DCMAKE_SKIP_RPATH=TRUE \
    -DCMAKE_BUILD_TYPE=$RELEASE ..
```

Reply:
Yes, I do have PACKAGE_MODE=ON set; I should try a make install with your pr to see how everything behaves

Reply:
Yes, I think PACKAGE_MODE=ON implies it will be installed system wide and probably won't work from the build directory or DESTDIR.

Reply:
Double checked and this doesn't change anything in my local setup, system wide seems to work too.